setup.py

```diff
@@ -19,9 +20,15 @@
     log_extras_require = [l.strip() for l in log_extras_requirements.readlines()]
 
 
+with open(os.path.join('bigflow', '_version.py'), 'r') as version_file:
```

Comment:
Why can't we just import bigflow._version?

Reply:
Because it will trigger loading of '__init__.py', which will try to import the whole BigFlow.

Reply:
So maybe it will be simpler to hold VERSION in setup.py?

Reply:
Then it's not available in the bigflow package.

Reply:
I vote for the Andrei solution.

Reply:
👍